_data/simple-icons.json

```diff
@@ -3870,7 +3871,7 @@
         {
             "title": "Hack The Box",
             "hex": "9FEF00",
-            "source": "https://www.hackthebox.eu/docs/Hack_The_Box_Brand_Assets_Guide.pdf",
+            "source": "https://www.hackthebox.eu/",
             "guidelines": "https://www.hackthebox.eu/docs/Hack_The_Box_Brand_Assets_Guide.pdf"
```

Comment:
I think both source and guidelines should be https://www.hackthebox.eu/newsroom/new-logo